### PR TITLE
App Tracer.app v1.7.1

### PR DESCRIPTION
**Pull request by gaberoo**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

---

Are there style guidelines for whether to rename the `app` bundle when it's versioned:
```
Tracer v1.7.1.app
```
Should this be renamed to `Tracer.app`?

### CODE REVIEW

File: Casks/tracer.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  # github.com/beast-dev/tracer was verified as official when first introduced to the cask
+  url "https://github.com/beast-dev/tracer/releases/download/v#{version}/Tracer.v#{version}.dmg"
+  appcast 'https://github.com/beast-dev/tracer/releases.atom'
+  name 'Tracer'
+  homepage 'https://beast.community/tracer'
+
+  app "Tracer v#{version}.app"
+
+  caveats do
+    depends_on_java
```
Comment (vitorgalvao) [inline Casks/tracer.rb]: There [needs to be a version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/caveats.md#caveats-mini-dsl), here.

**Comment by vitorgalvao** [comment]
> Should this be renamed to `Tracer.app`?

[No](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/app.md#target-should-only-be-used-in-select-cases).

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> The Cask is too obscure (for example, a self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 50 stars)).

**Comment by gaberoo** [comment]
> Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):
> 
> > The Cask is too obscure (for example, a self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 50 stars)).

Hi, hmm, OK, in the end it's up to you, I totally understand. I'm just a little surprised because `Tracer` is a necessary companion app to [beast](https://github.com/Homebrew/homebrew-core/blob/master/Formula/beast.rb), which is in `homebrew-core`. Also, another companion app, `figtree` is already in `homebrew-cask` (https://github.com/Homebrew/homebrew-cask/blob/master/Casks/figtree.rb).

**Comment by vitorgalvao** [comment]
@gaberoo Both of those examples (barely) pass the 50 stars threshold. I’d make an exception for this one on account of the other two, but neither of those is terribly popular. Each of them was installed less than 90 times in the last 90 days.

**Comment by gaberoo** [comment]
@vitorgalvao, OK, gotcha! Alternatively I'll see if we want to include this in `brewsci/bio`. I just figured that because of the `figtree` precedent it would be better housed here. Also, including `java` app bundles is just much more logical as a cask. Thanks for your input!

**Pull request closed without merging**